package.json

```diff
@@ -42,8 +42,9 @@
     "node-persist": "^2.1.0",
     "onoff": "^1.1.2",
     "openzwave-shared": "^1.3.2",
+    "request-promise": "^4.2.1",
     "serialport": "^4.0.7",
-    "sqlite3": "^3.1.8",
+    "sqlite3": "^3.1.9",
```

Comment:
Any particular reason for this change? @hobinjk Will this interfere with yarn at all? I'm unsure how that works.

Reply:
Yes, this change requires this PR to have `yarn install` run and the changes to `yarn.lock` committed. The current version of sqlite3 in the lockfile is `3.1.8`.

Reply:
I had a problem with sqlite 3.1.8 too and only had it fixed after update to the latest version.